pims/base_frames.py

```diff
@@ -33,6 +33,12 @@ def pixel_type(self):
         """Returns a numpy.dtype for the data type of the pixel values"""
         pass
 
+    @property
+    def dtype(self):
+        # The choice of using the separate name `pixel_type` was historical
+        # and needlessly made PIMS objects look less like numpy arrays.
+        return self.pixel_type
```

Comment:
Interesting point. Maybe we actually want `pixel_type` and `dtype` to be allowed to be different things in some cases?